APE24.rst

```diff
@@ -0,0 +1,477 @@
+   pyOpenSci will try to help the maintainer build a new maintainer team;
+2. Otherwise, pyOpenSci will gracefully sunset it from the list of maintained/accepted
+   packages.
+
+After sunsetting, if a package becomes active again, it is up to the package
+maintainer to contact Astropy or pyOpenSci in order to have it be actively
+listed once more. An example scenario that might happen is when the sole
+maintainer goes on a hiatus longer than the time frame set in
+`maintainer responsiveness <https://www.pyopensci.org/software-peer-review/our-process/policies.html#maintainer-responsiveness>`_
+and then comes back to a sunsetted package.
+
+.. _Scientific Python: https://scientific-python.org/
+
+Finding reviewers
+-----------------
+
+pyOpenSci currently has a list of reviewers who have signed up for this task using the
+`reviewer sign-up form`_.
+Because they utilize 2 reviewers for each package, they generally try to find
+a reviewer with domain-specific expertise, while the other with or without.
+Sometimes, the second reviewer will focus instead on general usability,
+ease of installation, documentation quality, or packaging infrastructure.
+For every review, they target a diversity of contributors to ensure that they
+have a mix of varying gender, cultural, etc., identities.
+
+If we decide to partner with pyOpenSci, our reviewers would sign up using
+the pyOpenSci form above, specifying:
+
+* astronomy domain expertise, and
+* membership in the Astropy community.
+
+pyOpenSci does not publicly list all the people who signed up
+(i.e, a reviewer is only public during the review process)
+but the `pyOpenSci Editorial Board`_ has access to the list.
+
+pyOpenSci will respect the Astropy-specific criteria for someone to be a
+reviewer for Affiliated request, namely:
+
+* familiarity with the Astropy project,
+* ability to judge whether a package integrates well with the Astropy ecosystem
+  (as per pre-APE 24 guidelines), and
+* having domain expertise in the area of the package (e.g., galaxy evolution).
+
+pyOpenSci and Astropy both ask editors/reviewers disclose any potential conflict of interest (COI)
+prior to agreeing to review a package. In the event where COI occurs, with this partnership,
+we would follow the
+`pyOpenSci COI process <https://www.pyopensci.org/software-peer-review/our-process/policies.html#conflict-of-interest-for-reviews-and-editors>`_.
+pyOpenSci invites the Astropy community to review this language to ensure it meets our needs.
+
+.. _reviewer sign-up form: https://forms.gle/GHfxvmS47nQFDcBM6
+
+Review turnaround time
+----------------------
+
+Astropy currently does not enforce any concrete turnaround time mainly because
+we do not know who ends up reviewing a given submission nor have control
+over their schedule. Furthermore, we only have two editors who may not have
+sufficient time themselves to actively push a review forward. Therefore, a review
+could take weeks, months, or (gasp) years.
```

Comment:
I think this makes it sound worse than it is. Most reviews come in within weeks (after prodding by the editors). If not, the editors need to find a different reviewer. That's parallel to any peer review process. Even "nature" has no control over the peer reviewer's time and they still say "2 weeks" turn-around. When a review lingers, it's usually because a package does not respond in a timely manner to reviewer' requests for changes. Even that's not necessarily bad (e.g. if the reviewer suggests a big feature that they decide to include as part of their normal development process), see  e.g. https://github.com/astropy/astropy.github.com/pull/491 for an example.

If we want to be "faster" we could just close those PRs and then claim that "every package is decided within 4 week". But what would we gain by doing that?

Reply:
I wonder if stricter initial triaging would have weeded out the ones that dragged on, but it is neither here nor there. I accepted your text changes for this, so marking as resolved. Thanks!